Chandra/Time/Time.py

```diff
@@ -329,6 +336,15 @@ def mx_DateTime_ISO_ParseDateTime(t):
         raise ValueError('mxDateTime format is unavailable, ask TLA for migration options')
 
 
+def match_func_cxotime(expr, time_in):
+    # Access attributes that strongly indicate this is an astropy.time.Time object
+    time_in.light_travel_time
+    time_in.sidereal_time
+    time_in.format
+    time_in.shape
+    return str(time_in.secs)
```

Comment:
Not that I am opposed, but why do all these checks?

If you want to check that this is an astropy.time.Time instance, you can use isinstance. If what you want to check is the behavior, then isn't `time_in.secs` enough?

Reply:
The idea is to be extremely sure the object is an astropy Time subclass without actually importing astropy.time.Time.  Just looking for a `secs` attribute is not strong enough because there are many unrelated classes that have a `secs` attribute. However almost no random classes will have all those attributes.

That is indeed a bit adhoc / hacky. Maybe there is a better way, like digging through `.__class__.__mro__` to find `'astropy.time.core.Time'`. I think that works, as long as astropy never refactors or changes the module name `core.py`.

Reply:
That reminds me the intent was to be slightly sneaky and actually use `time_in.cxcsec` to allow initialization from any astropy `Time` subclass since `cxcsec` is built-in. For initializing `DateTime` any `Time` object should be sufficient, but going the other way will result in a `CxoTime` coming out.

Reply:
Ah.  Seems like both those thoughts ('without importing `astropy.time` and `allow initialization from any subclass`) could just make their way into the comment for now.

Reply:
But since this was a WIP...

Reply:
I removed the WIP. I need to check again that testing coverage is OK but I don't have any outstanding items for implementation in mind.